src/main/java/cubicchunks/server/chunkio/IONbtReader.java

```diff
@@ -114,6 +118,11 @@ static Cube readCubeAsyncPart(Column column, final int cubeX, final int cubeY, f
     }
 
     static void readCubeSyncPart(Cube cube, ICubicWorldServer world, NBTTagCompound nbt) {
+        // To preserve compatibility with previous version of CubicChunks.
```

Comment:
We don't care about compatibility yet, no need to be able to load worlds from any previous versions. That's the main reason it's not released - so I don't need to keep a lot of code just to support worlds from older versions.